lib/Yield/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+revng_add_library_internal(
+  revngYield
+  SHARED
+  Assembly/DisassemblerInstance.cpp)
+
+set(DISASSEMBLER_ARCHITECTURE_SUPPORT "")
+foreach(CURRENT_ARCHITECTURE AArch64 AMDGPU ARM Mips SystemZ X86)
```

Comment:
We don't really support `AMDGPU` but OK.

Reply:
This snipped is straight up copied from caliban (Maybe I renamed a variable or something, I don't remember), so we have always supported AMDGPU ;) As far as disassembly concerned that is.

Reply:
Done.